packages/@aws-cdk/aws-cognito-identitypool-alpha/lib/identitypool.ts

```diff
@@ -155,7 +157,8 @@ export class IdentityPoolProviderUrl {
   }
 
   /** User Pool Provider Url */
-  public static userPool(url: string): IdentityPoolProviderUrl {
+  public static userPool(userPool: UserPool, userPoolClient: UserPoolClient): IdentityPoolProviderUrl {
```

Comment:
I know we're ok with breaking changes, but this seems like it would ruin anyone who was simply supplying a url here that was not connected to a specific `UserPool` or `UserPoolCliet` construct. I'm not sure what the difference would be if the `IdentityPoolProviderType` turns from `USER_POOL` to `CUSTOM`

Reply:
`IdentityPoolProviderType` (`IdentityPoolProviderUrl.type`) is not used since #16190 (implemented PR).
Changing `.userPool(somthing)` to `.custom(something)` makes no differences in CloudFormation template.